spikeinterface/curation/auto_merge.py

```diff
@@ -135,7 +135,7 @@ def get_potential_auto_merge(
 
     # STEP 2 : remove contaminated auto corr
     if 'remove_contaminated' in steps:
-        nb_violations, contaminations = compute_refrac_period_violations(we, refractory_period_ms=refractory_period_ms,
+        contaminations, nb_violations = compute_refrac_period_violations(we, refractory_period_ms=refractory_period_ms,
```

Comment:
incredible!!! good catch.
@DradeAW : we haven't see it !!!!!

Reply:
I changed it in the QM refactor PR

Reply:
ok